paimon-spark/paimon-spark-common/pom.xml

```diff
@@ -127,10 +127,6 @@ under the License.
                     <groupId>org.apache.orc</groupId>
                     <artifactId>orc-core</artifactId>
                 </exclusion>
-                <exclusion>
-                    <groupId>org.apache.orc</groupId>
-                    <artifactId>orc-mapreduce</artifactId>
-                </exclusion>
```

Comment:
do we need orc-mapreduce here?

Reply:
spark write orc need it, and it is in test scope